### PR TITLE
ADEN-11327 revert liveramp

### PR DESCRIPTION
**Pull request by jakubbanas**

### CODE REVIEW

File: src/ad-bidders/prebid/index.ts [hunk 1]
```diff
@@ -47,15 +49,20 @@ export class PrebidProvider extends BidderProvider {
...
 
 		this.isLazyLoadingEnabled = this.bidderConfig.lazyLoadingEnabled;
+		this.adUnits = setupAdUnits(this.isLazyLoadingEnabled ? 'pre' : 'off');
+		this.bidsRefreshing = context.get('bidders.prebid.bidsRefreshing') || {};
+		this.isATSAnalyticsEnabled = context.get('bidders.liveRampATSAnalytics.enabled');
+
 		this.prebidConfig = {
 			debug: ['1', 'true'].includes(utils.queryString.get('pbjs_debug')),
-			enableSendAllBids: true,
+			enableSendAllBids: !!context.get('bidders.prebid.sendAllBids'),
```
Comment (Tomasz-at-Fandom) [inline src/ad-bidders/prebid/index.ts:65]: Do we want this back? I don't see this being set anywhere.

Comment (Tomasz-at-Fandom) [reply src/ad-bidders/prebid/index.ts:65]: @dmnsobczak removed this only recently as part of ADEN-11248, are we certain it is related?

Comment (Tomasz-at-Fandom) [reply src/ad-bidders/prebid/index.ts:65]: This is also being called on line 236.

Comment (jakubbanas) [reply src/ad-bidders/prebid/index.ts:65]: Good point, let me check that.

**Review by Tomasz-at-Fandom — APPROVED** [review]

**Pull request merged**